vllm/v1/core/kv_cache_manager.py

```diff
@@ -9,6 +9,139 @@
 logger = init_logger(__name__)
 
 
+@dataclass
```

Comment:
QQ: Can we make this file shorter by moving some classes (or methods) like `KVCacheBlock` and `FreeKVCacheBlockQueue` to a separate file?

Reply:
Yeah I was thinking about this too, but have no idea what the file name should be. Any thoughts?